desc/plotting.py

```diff
@@ -1,3 +1,4 @@
+from turtle import color
```

Comment:
is this used?

Reply:
nope I'll remove, not sure why it's there